library/keystone_user.py

```diff
@@ -118,7 +118,8 @@ def authenticate(endpoint, token, login_user, login_password, login_tenant_name)
         return client.Client(endpoint=endpoint, token=token)
     else:
         return client.Client(auth_url=endpoint, username=login_user,
-                             password=login_password, tenant_name=login_tenant_name)
+                             password=login_password, tenant_name=login_tenant_name,
+                             cacert="/opt/stack/ssl/openstack.crt")
```

Comment:
I don't see why this is needed. Our CI system generates a cert every time that's self signed, and doesn't run into any issues with this.